dbbackup/storage/builtin_django.py

```diff
@@ -0,0 +1,29 @@
+"""
+Lazy storage adapted to Django buit-in one.
+"""
+from django.core.files.storage import get_storage_class
+from .base import BaseStorage
+from dbbackup import settings
```

Comment:
The relative and absolute imports right next to each other look funny, sorry about nit-picking :)

I prefer relative imports.

    from .. import settings